include/vga.h

```diff
@@ -923,75 +869,93 @@ struct VGA_Dac {
 	uint8_t read_index  = 0;
 };
 
-struct VGA_SVGA {
+struct VgaSvga {
 	Bitu readStart = 0, writeStart = 0;
-	Bitu bankMask = 0;
-	Bitu bank_read_full = 0;
+	Bitu bankMask        = 0;
+	Bitu bank_read_full  = 0;
 	Bitu bank_write_full = 0;
-	uint8_t bank_read = 0;
-	uint8_t bank_write = 0;
-	Bitu bank_size = 0;
+	uint8_t bank_read    = 0;
+	uint8_t bank_write   = 0;
+	Bitu bank_size       = 0;
 };
 
-union VGA_Latch {
+union VgaLatch {
 	uint32_t d = 0;
 	uint8_t b[4];
 };
 
-struct VGA_Memory {
+struct VgaMemory {
 	uint8_t* linear = {};
 };
 
 struct VGA_Changes {
-	//Add a few more just to be safe
-	uint8_t *map = nullptr; /* allocated dynamically: [(VGA_MEMORY >> VGA_CHANGE_SHIFT) + 32] */
-	uint8_t checkMask = 0;
-	uint8_t frame = 0;
-	uint8_t writeMask = 0;
-	bool active = 0;
-	uint32_t clearMask = 0;
-	uint32_t start = 0, last = 0;
+	// Add a few more just to be safe
+	// Allocated dynamically: [(VGA_MEMORY >> VGA_CHANGE_SHIFT) + 32]
+	uint8_t* map = nullptr;
+
+	uint8_t checkMask    = 0;
+	uint8_t frame        = 0;
+	uint8_t writeMask    = 0;
+	bool active          = 0;
+	uint32_t clearMask   = 0;
+	uint32_t start       = 0;
+	uint32_t last        = 0;
 	uint32_t lastAddress = 0;
 };
 
-struct VGA_LFB {
+struct VgaLfb {
 	uint32_t page = 0;
 	uint32_t addr = 0;
 	uint32_t mask = 0;
-	PageHandler *handler = nullptr;
+
+	PageHandler* handler = nullptr;
 };
 
-struct VGA_Type {
+struct VgaType {
 	VGAModes mode = {}; /* The mode the vga system is in */
-	uint8_t misc_output = 0;
-	VGA_Draw draw = {};
-	VGA_Config config = {};
-	VGA_Internal internal = {};
+
+	uint8_t misc_output  = 0;
+	VgaDraw draw         = {};
+	VgaConfig config     = {};
+	VgaInternal internal = {};
+
 	/* Internal module groups */
-	VGA_Seq seq = {};
-	VGA_Attr attr = {};
-	VGA_Crtc crtc = {};
-	VGA_Gfx gfx = {};
-	VGA_Dac dac = {};
-	VGA_Latch latch = {};
-	VGA_S3 s3 = {};
-	VGA_SVGA svga = {};
-	VGA_HERC herc = {};
-	VGA_TANDY tandy = {};
-	VGA_OTHER other = {};
-	VGA_Memory mem = {};
-	// this is assumed to be power of 2
+	VgaSeq seq     = {};
+	VgaAttr attr   = {};
+	VgaCrtc crtc   = {};
+	VgaGfx gfx     = {};
+	VgaDac dac     = {};
+	VgaLatch latch = {};
+	VgaS3 s3       = {};
+	VgaSvga svga   = {};
+	VgaHerc herc   = {};
+	VgaTandy tandy = {};
+	VgaOther other = {};
+	VgaMemory mem  = {};
+
+	// This is assumed to be power of 2
 	uint32_t vmemwrap = 0;
-	 // memory for fast (usually 16-colour) rendering,
-	 // always twice as big as vmemsize
+
+	// Memory for fast (usually 16-colour) rendering,
+	// always twice as big as vmemsize
+	//
 	uint8_t* fastmem  = {};
 	uint32_t vmemsize = 0;
+
 #ifdef VGA_KEEP_CHANGES
-	VGA_Changes changes = {};
+	VgaChanges changes = {};
 #endif
-	VGA_LFB lfb = {};
+
+	VgaLfb lfb = {};
+
 	// Composite video mode parameters
-	int ri = 0, rq = 0, gi = 0, gq = 0, bi = 0, bq = 0;
+	// TODO enclose in struct
+	int ri        = 0;
```

Comment:
Would it be feasible to change `int`s to some type of fixed size, like `int32_t`?

Reply:
I've done that plus some other minor cleanup. But note I'm only touching these files accidentally because of the renames—I'm not actually working on these other files.